chocolatey-diff/Public/Get-FullPackageInfo.ps1

```diff
@@ -19,24 +19,30 @@ function Get-FullPackageInfo {
         [string] $PackageId
     )
 
-    $pkgInfo = @{
-        Id       = $PackageId
-        Url      = "$(Get-ChocolateyPackageGalleryUrl)/${$PackageId}"
-        Versions = @()
-    }
+    if (-not $script:pkgInfo -or $script:pkgInfo.Id -notmatch $PackageId) {
```

Comment:
I'd love to see some `Write-Verbose` to tell if these conditions match or not.
(a.k.a. tell the user if the package info is fetched from the repository or taken from the cache)

Reply:
I have added some extra details.

Reply:
Should maybe have a cache timeout as well, but that is probably more of an improvement request at a later time.